### PR TITLE
stub out quickpr and point to published docs

### PR DESCRIPTION
**Pull request by samccann**
This content moved to docs.ansible.com

### CODE REVIEW

File: create_pr_quick_start_guide.rst [hunk 1]
```diff
@@ -2,268 +2,4 @@
...
-
-You will see the CI status in the bottom of your pull request.
-If they are green and you think that you do not want to add more commits before someone should take a closer look at it, remove ``[WIP]`` from the title. Mention the issue reporter in a comment and let contributors know that the pull request is "Ready for review".
-
-24. Wait for reviews. You can also ask for review in the ``#ansible-community`` `Matrix/Libera.Chat IRC channel <https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat>`_.
-
-25. If the pull request looks good to the community, committers will merge it.
-
-For details, refer to the `Ansible developer guide <https://docs.ansible.com/ansible/latest/dev_guide/index.html>`_.
+This guide has moved to `https://docs.ansible.com/ansible/devel/community/create_pr_quick_start.html`_.
```
Comment (Andersson007) [inline create_pr_quick_start_guide.rst:5]:
```suggestion
This guide has moved to `docs.ansible.com <https://docs.ansible.com/ansible/devel/community/create_pr_quick_start.html>`_.
```

**Review by Andersson007 — COMMENTED** [review]
@samccann thanks for doing this!

**Comment by samccann** [comment]
ready_for_review

**Review by Andersson007 — APPROVED** [review]
LGTM, thanks @samccann !

**Pull request merged**